docs/docs/gatsby-cli.md

```diff
@@ -41,12 +38,18 @@ development server.
 | `-o`, `--open`  | Open the site in your (default) browser for you |
 | `-S`, `--https` | Use HTTPS                                       |
 
+For example `gatsby develop --port 4000`.
```

Comment:
Probably drop these. We don't need to document how to pass options, we just need to document that options exist!

Reply:
This line in or out?

When running Gatsby CLI commands with options using NPM scripts you need to include `--` to forward the options.